research/css/index.css

```diff
@@ -35,41 +32,21 @@ body {
     margin: 2rem 0;
   }
   
-  /* Main marketing message and sign up button */
-  .jumbotron {
+  .card {
```

Comment:
Modifying library class's style directly is not suggested. Try to make another class for your own style, and overlap on your desired tag. Not only these, `header`, `footer`, `container` may be included.